.github/scripts/test_trymerge.py

```diff
@@ -490,6 +492,137 @@ def test_pr_changed_submodule_detection(self, *args: Any) -> None:
         self.assertEqual(pr.get_changed_submodules(), ["third_party/kineto"])
         self.assertFalse(pr.has_invalid_submodule_updates())
 
+    def test_remove_job_name_suffix(self, *args: Any) -> None:
```

Comment:
You don't really need args here, do you?
```suggestion
    def test_remove_job_name_suffix(self) -> None:
```

Reply:
Ah, this function doesn't need to use `arg*` but still needs to declare it as the test class itself has 3 `mock.patch`

```
@mock.patch("trymerge.read_flaky_rules", side_effect=empty_flaky_rules)
@mock.patch("trymerge.get_rockset_results", side_effect=empty_rockset_results)
@mock.patch("trymerge.gh_graphql", side_effect=mocked_gh_graphql)
class TestTryMerge(TestCase):
    ...
```

So I think all the test cases need at least 4 arguments.